common/utilities.ts

```diff
@@ -39,6 +39,11 @@ export const isValidPassword = (password) => {
   return C.regex.password.test(password);
 };
 
+// add password confirmation
+export const isValidconfirmPassword = (confirmPassword) => {
```

Comment:
nit: This isn't necessary either, it can use the valid password test. The only reason to keep this method is if you're comparing the old one to the new one (and making sure they are valid) but you can still use the same regex.

Also, `isValidConfirmPassword`

Reply:
Removed the duplicate logic to use isValidPassword. 

Thank you for catching the camel case miss for isValidConfirmPassword.